spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -31,13 +47,166 @@
 @RunWith(MockitoJUnitRunner.class)
```

Comment:
Please add your name above, as the file now mostly your code anyway. :wink:

Reply:
Done.